tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -22,8 +22,10 @@ Feature: spaces management
       | id     |
       | team.a |
     And "Alice" updates the quota for space "team.a" to "50"
-    And "Alice" disables the space "team.a"
-    And "Alice" deletes the space "team.a"
+    And "Alice" disables the space "team.a" using the context-menu
+    And "Alice" deletes the space "team.a" using the context-menu
+    And "Alice" disables the space "team.b" using the context-menu
+    And "Alice" deletes the space "team.b" using the context-menu
```

Comment:
This is supposed to be handled by Brian (see below). Please either remove those 2 lines, or add a comment to remove this as soon as the code below gets commented it.